webdriver/tests/element_send_keys/send_keys.py

```diff
@@ -71,3 +71,41 @@ def test_stale_element(session, inline):
 
     response = element_send_keys(session, element, "foo")
     assert_error(response, "stale element reference")
+
+@pytest.mark.parametrize("value",[
+    [u'\ue018', ";"],
+    [u'\ue019', "="],
+    [u'\ue024', "*"],
+    [u'\ue025', "+"],
+    [u'\ue027', "-"],
+    [u'\ue028', "."],
+    [u'\ue029', "/"],
+    [u'\ue01a', "0"],
+    [u'\ue023', "9"]
+    ])
+def test_printable_normalised_key_value(session, inline, value):
+    # TODO This is not extensive list due to timeouts from wpt if there are a lot of test
+    # See https://github.com/web-platform-tests/wpt/issues/32899
```

Comment:
This issue is no longer needed I assume and we can close?

Reply:
I still hit this issue in Chrome.

Reply:
I parameterized the bottom test case in the file and that hits it on my local machine with chrome.